google-cloud-logging/test/google/cloud/logging/project/list_entries_test.rb

```diff
@@ -224,7 +224,7 @@
     list_res = OpenStruct.new(page: OpenStruct.new(response: Google::Logging::V2::ListLogEntriesResponse.decode_json(list_entries_json(3, "next_page_token"))))
 
     mock = Minitest::Mock.new
-    mock.expect :list_log_entries, list_res, [["project1"], resource_names: nil, filter: nil, order_by: nil, page_size: nil, options: default_options]
+    mock.expect :list_log_entries, list_res, [[], project_ids: ["project1"], filter: nil, order_by: nil, page_size: nil, options: default_options]
```

Comment:
Oh wait, I see what you are asking. Before we only had one array, and we would make sure it had a default value. Now we can specify both resource_names _or_ project_ids. So if neither are proven fed we will default resource_names to the current project resource. But if projects are specified we don't apply that default.